README.md

```diff
@@ -21,25 +21,27 @@
 
 ## What is apisix-control-plane
 
-apisix-control-plane is an implementation of providing a control plane for DPs (Data Plane) like apache apisix or other gateways. with a `yml` configuration capability, We can use `yml` to define the behavior of any DP.
+apisix-control-plane is an implementation of providing a control plane for DPs (Data Plane) like Apache APISIX or other gateways. with a `yaml` configuration capability, We can use `yaml` to define the behavior of any DP.
 
 ## Why do we need to implement a control-plane
 
-for now, we have a default implementation for apache APISIX.
+For now, we have a default implementation for Apache APISIX.
 
-As we knows, apache APISIX is a high-performance gateway. When using APISIX we need to inform apisix of some proxy rules, and distribute these rules to apisix modules. We can call it contol-plane, similar to pilot in istio. Of course, the distribution configuration is only the most basic function of the control plane.
+As we knows, Apache APISIX is a high-performance gateway. When using APISIX we need to inform apisix of some proxy rules, and distribute these rules to apisix modules. We can call it contol-plane, similar to pilot in istio. Of course, the distribution configuration is only the most basic function of the control plane.
```

Comment:
`When using APISIX we need to inform apisix`

Please pay attention to "apisix", they are wrong styles.

Reply:
done